unit_tests/unit_test_data/saved_derived_Original.amlg

```diff
@@ -7663,7 +7663,7 @@
 																	context_values react_context_values
 																	weight_feature weight_feature
 																	leave_case_out leave_case_out
-																	extra_audit_features extra_audit_features
```

Comment:
I don't think we should change the names of the parameters here in 'saved_derived_Original.amlg'. 

I don't think it would break anything, but this is supposed to represent an older version of a Trainee